src/alexer.mll

```diff
@@ -93,14 +93,13 @@ and token = parse
   | (numeral+ as i)
       { [ NUM (i, "") ] }
   | ("\\b" ['0' '1']+ as b)
-      { Bytes.set b 0 '0' ;
+      { let b = "0" ^ (String.sub b 1 ((String.length b) - 1)) in
```

Comment:
If you move the open parenthesis on the previous line to make it:
```ocaml
  | "\\b" (['0' '1']+ as b)
```
then you can change this line like this:
```suggestion
      { let b = "0b" ^ b in
```